packages/storybook/src/nimble/icon-base/icons.stories.ts

```diff
@@ -23,7 +23,9 @@ import {
 type IconName = keyof typeof nimbleIconComponentsMap;
 const data = Object.values(nimbleIconComponentsMap).map(iconClass => ({
     tag: DesignSystem.tagFor(iconClass),
-    metaphor: iconMetadata[iconClass.name as IconName].tags.join(', ')
+    metaphor: iconMetadata[iconClass.name as IconName]?.tags
+        ? iconMetadata[iconClass.name as IconName].tags.join(', ')
+        : 'Metaphor not found. Likely a build minification issue.'
```

Comment:
don't think this change is needed anymore. If it stays around should  be more aggressive like throw an error

Reply:
Updated the pattern to remove the need for referencing the `ClassName.name` altogether. Should be more robust against minifiers generally.